pkg/shell/state.tsx

```diff
@@ -646,6 +647,21 @@ export class ShellState extends EventEmitter<ShellStateEvents> {
         this.current_manifest_item = item;
         this.current_manifest = compiled.find_path_manifest(location.path);
 
+        this.current_fullscreen = false;
+        if (this.current_manifest._fullscreen) {
+            const fs = this.current_manifest._fullscreen;
+            const path_prefix = location.path.split("/", 1)[0];
+            for (const path_suffix in fs) {
+                if ((path_suffix == "" && path_prefix == location.path) ||
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-21670-c1859e10-20250731-141019-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)